include/exadg/utilities/print_general_infos.h

```diff
@@ -93,6 +94,17 @@ print_dealii_info(dealii::ConditionalOStream const & pcout)
   // clang-format on
 }
 
+// print ExaDG info
+inline void
+print_exadg_info(dealii::ConditionalOStream const & pcout)
+{
+  pcout << std::endl
+        << "ExaDG info:" << std::endl
+        << std::endl
+        << "  ExaDG git version " << GitRevision::get_exadg_git_shortrev() << " on branch "
```

Comment:
We should discuss why this is done differently than for deal.II a few lines above. It is not clear to me why we introduce `exadg_git_info.h`, `exadg_git_info.cpp` in this PR? One solution could be to change it on the deal.II side (with changes in the deal.II library itself and/or get-functions in ExaDG).

Reply:
I would like to include the file which contains the #defines (currently strings, but I will change it) in one source file only, to avoid recompilation of several files if the commit hash changes. Therefore, I introduced the get functions. It would be good to encapsulate the files for deal.ii as well, but it is not as important since the deal.ii library is probably less often recompiled if the focus is on developing code for ExaDG. Personally, I don't think this needs a change on the deal.ii side. 

I will update this PR as soon as possible such that it could be merged